packages/devtools_app/lib/src/screens/deep_link_validation/deep_links_controller.dart

```diff
@@ -415,20 +414,26 @@ class DeepLinksController extends DisposableController {
   }
 
   Future<void> validateLinks() async {
-    List<LinkData> linkdata = _allRawLinkDatas;
-    if (linkdata.isEmpty) {
+    // List<LinkData> linkdata = _allRawLinkDatas;
+    final appLinkSettings = _androidAppLinks[selectedVariantIndex.value];
+    if (appLinkSettings == null) {
       pagePhase.value = PagePhase.noLinks;
       return;
     }
+    if (appLinkSettings.error != null) {
+      pagePhase.value = PagePhase.analyzeErrorPage;
+      return;
+    }
     pagePhase.value = PagePhase.linksValidating;
+    List<LinkData> linkdata = _allRawLinkDatas(appLinkSettings);
```

Comment:
why are we setting this and then immediately setting it to something else on the next line?